core/src/main/java/feast/core/config/JobConfig.java

```diff
@@ -64,6 +68,16 @@
         .build();
   }
 
+  @Bean
+  public JobGroupingStrategy getJobGroupingStrategy(
+      FeastProperties feastProperties, JobRepository jobRepository) {
+    if (feastProperties.getJobs().getConsolidateJobsPerSource()) {
```

Comment:
Small nitpick, for readability we are trying to pop out the nested config into separate variables instead of referencing it directly. So

```
String shouldConsolidateJobsPerSource = feastProperties.getJobs().getConsolidateJobsPerSource();

if (shouldConsolidateJobsPerSource){...}
```